dashboard/db/migrate/20150804174622_migrate_phase_values.rb

```diff
@@ -0,0 +1,34 @@
+class MigratePhaseValues < ActiveRecord::Migration
+  PHASES = {
+      1 => 'Phase 1',
+      2 => 'Phase 2',
+      3 => 'Phase 2 Online',
+      4 => 'Phase 3A',
+      5 => 'Phase 3B',
+      6 => 'Phase 3C',
+      7 => 'Phase 3D',
+      8 => 'Phase 4'
+  }
+  def up
+    ActiveRecord::Base.transaction do
+      Workshop.find_each do |workshop|
+        phase = workshop.phase
+        if PHASES.invert[phase]
+          workshop.phase = PHASES.invert[phase]
+        else
+          raise "error"
```

Comment:
Please change the string to something that will make it clear what the problem is and how to fix it.

In this case, something like 'Unknown phase name #{phase}.  Please update the PHASES map.'